02-Homework/script.js

```diff
@@ -0,0 +1,56 @@
+// Assignment Code
+var generateBtn = document.querySelector('#generate');
+
+function generatePassword() {
+  // PLEASE ADD ALL YOUR CODE HERE
+}
+alert ("text");
+var text
```

Comment:
What are these variables used for? They haven't been assigned a value so currently they will both return `undefined`.

Reply:
Also - please refrain from using `var` to define variables. We should be using `const` or `let` if the variable needs to be redefined at any point.